### PR TITLE
Face Recognition using LBP features and Support vector machine

### PR DESCRIPTION
**Pull request by Bhagyashri2000**
Face detection and recognition using LBP features and Support Vector Machine.
(LBP features- LBPs compute a local representation of texture. This local representation is constructed by comparing each pixel with its surrounding neighborhood of pixels)
This model first creates the rectangle and crop the facial area from image then extracts LBP features from image and pass it through SVM.

### CODE REVIEW

File: Facial_Recognition_Models/Recognition Using LBP_SVM/Readme.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+## Dependencies
+    pip install numpy
+    pip install opencv-python
+    pip install sklearn
+    pip install skimage
+    
+    
+## Quick Start
+    1] git clone https://github.com/akshitagupta15june/Face-X.git
```
Comment (SanjayMarreddi) [inline Facial_Recognition_Models/Recognition Using LBP_SVM/Readme.md]: Change this accordingly to the current project.

File: Facial_Recognition_Models/Recognition Using LBP_SVM/Readme.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    2] cd Recognition using LBP_SVM
```
Comment (SanjayMarreddi) [inline Facial_Recognition_Models/Recognition Using LBP_SVM/Readme.md]: Same here, Change this accordingly to the current project Directory

**Review by SanjayMarreddi — CHANGES_REQUESTED** [review]
@Bhagyashri2000 Everything looks fine, Just do the below changes

File: Facial_Recognition_Models/Recognition Using LBP_SVM/Readme.md (hunk 2 above)
Comment (Bhagyashri2000) [reply Facial_Recognition_Models/Recognition Using LBP_SVM/Readme.md]: @SanjayMarreddi Updated readme file.

**Review by SanjayMarreddi — APPROVED** [review]

**Pull request merged**